addons/event/static/src/client_action/event_registration_summary_dialog.js

```diff
@@ -73,6 +74,21 @@ export class EventRegistrationSummaryDialog extends Component {
     async onRegistrationConfirm() {
         await this.orm.call("event.registration", "action_set_done", [this.registration.id]);
         this.registrationStatus.value = "confirmed_registration";
+        this.props.close();
+        if (this.props.model) {
+            this.props.model.load();
+        }
+        if (this.props.doNextScan) {
+            this.onScanNext();
+        }
+    }
+
+    async undoRegistration() {
+        await this.orm.call("event.registration", "action_confirm", [this.registration.id]);
```

Comment:
Just wondering, reading this: no update needed on the registrationStatus ?

Reply:
Nope, `registrationStatus` is only used to display a message when user is marked. It doesn't matter if we update it or not.

Reply:
You mean, as we close the dialog, it does not make sense updating its content? Make sense